### PR TITLE
[TRAFODION-1839] Trafodion Installer Evolution

### PR DESCRIPTION
**Pull request by mkby**
new python version installer

### CODE REVIEW

File: install/python-installer/traf_dep.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+import re
+import os
+import sys
+import json
+import platform
+from common import run_cmd, cmd_output, err
+
+# not used
+EPEL_REPO = """
```
Comment (svarnau) [inline install/python-installer/traf_dep.py]: This should not need to be hard-coded. Doing a yum install of "epel-release" package will get you the epel repo file.

File: install/python-installer/traf_package.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    dbcfgs = json.loads(dbcfgs_json)
+
+    TRAF_DIR = '%s-%s' % (dbcfgs['traf_basename'], dbcfgs['traf_version'])
+
+    # untar traf package
+    TRAF_PACKAGE_FILE = '/tmp/' + dbcfgs['traf_package'].split('/')[-1]
+    run_cmd('mkdir -p %s' % TRAF_DIR)
+    run_cmd('tar xf %s -C %s' % (TRAF_PACKAGE_FILE, TRAF_DIR))
+
+    print 'Trafodion package uncompressed successfully!'
```
Comment (svarnau) [inline install/python-installer/traf_package.py]: I think "extracted" would make more sense to me than "uncompressed". But opinions may vary.

File: install/python-installer/traf_dep.py (hunk 1 above)
Comment (mkby) [inline install/python-installer/traf_dep.py]: we only need 'pdsh' dependency from epel repository, so I'm thinking to install 'pdsh' rpm directly, but not use epel repository. Is it acceptable?

Comment (svarnau) [inline install/python-installer/traf_dep.py]: Directly? I'm not sure what is meant, but I think it is best to install from epel repo.

Comment (gtapper) [inline install/python-installer/traf_dep.py]: I was wondering the same. Directly seems to imply that we're distributing the pdsh package?

**Comment by traflm** [comment]
A huge change, need sometime to review it. Is previous installer still work together with this new one? I mean, one can still use the old installer as well this new installer? Or it is a replacement abruptly?

**Comment by mkby** [comment]
The two installer will coexist for a short of time, both of them can be used. Once the new one is stable and acceptable by users, we can remove the old one. There're still features need to be implemented for the new installer, for example, elasticity.

**Comment by amandamoran** [comment]
+1 to Ming. Would also like a chance to review but if we are not making switch yet that's different.

**Comment by traflm** [comment]
is this good to be merged? 
I cannot have any comments, would let it be tested.

**Pull request merged**